recbole/properties/overall.yaml

```
@@ -7,8 +7,8 @@ reproducibility: True
 data_path: 'dataset/'
 checkpoint_dir: 'saved'
 show_progress: True
-save_dataset: False
-save_dataloaders: False
+save_dataset: True
+save_dataloaders: True
```

```suggestion
save_dataset: False
save_dataloaders: False
```
We don't recommend to set these parameters to `True` for disk space cost.